pkg/util/utils_unsupported.go

```diff
@@ -10,3 +10,8 @@ import (
 func FindDeviceNodes() (map[string]string, error) {
 	return nil, errors.Errorf("not supported on non-Linux OSes")
 }
+
+// CheckRootlessUIDRange is not implemented anywhere except Linux.
+func CheckRootlessUIDRange(uid int) error {
+	return errors.Errorf("this function is not supported on this os")
```

Comment:
I think this should return nil. If this is called by podman-remote, we don't want the check being done.